.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: node_js
+sudo: required
+env:
+  global:
+    - firebase_bucket=gs://dracco1993-tba-dev.appspot.com
+
+# branches:
+#   only:
+#   - master
```

Comment:
We'll probably want to change this after if gets merged, so that we're only doing it for `master`.